ipa.go

```diff
@@ -69,16 +72,6 @@ func FromBytes(fr *Fr, data []byte) {
 	fr.SetBytes(aligned[:])
 }
 
-func StemFromBytes(fr *Fr, data []byte) {
-	bytes := make([]byte, len(data))
-	copy(bytes, data)
-	fr.SetBytesLE(bytes)
-}
```

Comment:
This method is doing the same as `FromLEBytes`, thus I don't think it's worth having it.
Pushbacks accepted!

Reply:
Uhm, yeah. Honestly, I'd prefer to keep then `StemFromBytes`, at least add the `len(data) == StemSize` and call `FromLEBytes`.

That sounds better for sure since:
- We include that length check that you mention, and stop relying on callers doing the right thing.
- We avoid the code duplication that was annoying me, since `FromLEBytes` is correct.

I'm going to change to go that route... I'd prefer to be safe as much as possible.

Reply:
Done!